lib/text/vtt_text_parser.js

```diff
@@ -439,6 +440,77 @@ shaka.text.VttTextParser = class {
     }
   }
 
+  /**
+   * Converts voice style tag to be valid for xml parsing
+   * For example,
+   * input: <v Shaka>Test
+   * output: <v.voice-Shaka>Test</v.voice-Shaka>
+   *
+   * @param {string} payload
+   * @return {string} processed payload
+   * @private
+   */
+  static replaceVoiceStylePayload_(payload) {
+    const voiceTag = 'v';
+    const names = [];
+    let nameStart = -1;
+    let newPayload = '';
+    let hasVoiceEndTag = false;
+    for (let i = 0; i < payload.length; i++) {
+      if (payload[i] === '/') {
+        const end = payload.indexOf('>', i);
+        if (end <= i) {
```

Comment:
In what condition do we expect `end <= i`? We start searching at `i`, so `end` will only be less than `i` if it's -1. And we know that `payload[i]` is equal to `'/'`, so `end` is guaranteed to not equal `i`.

If this is just meant to handle `end` being -1, then it'd be more informative to write that out.

Reply:
Ok, I'll change it!

Reply:
Done!